cli/pkg/kctrl/cmd/package/init/init.go

```diff
@@ -132,7 +135,7 @@ func (o *InitOptions) Run() error {
 	if err != nil {
 		return err
 	}
-
+	o.ui.PrintHeaderText("Output")
```

Comment:
Was this being done someplace else before?

Reply:
Yup, just moving it down closer to the text